lib/internal/bootstrap/pre_execution.js

```diff
@@ -237,9 +237,16 @@ function setupInspectorHooks() {
 // this is used to deprecate APIs implemented in C++ where the deprecation
 // utitlities are not easily accessible.
 function initializeDeprecations() {
-  const { deprecate } = require('internal/util');
+  const { deprecate, getSystemErrorName } = require('internal/util');
   const pendingDeprecation = getOptionValue('--pending-deprecation');
 
+  process.binding('uv').errname = deprecate(getSystemErrorName,
+                                            'Directly calling process' +
+                                            '.binding(\'uv\').errname(<val>) ' +
+                                            'is deprecated. Please use ' +
+                                            'util.getSystemErrorName() ' +
+                                            'instead.', 'DEP0119');
```

Comment:
Everything bunched up at the end makes this a bit difficult to read... perhaps instead:

```js
  process.binding('uv').errname = deprecate(
    getSystemErrorName,
    'Directly calling process.binding(\'uv\').errname(<val>) ' +
    'is deprecated. Please use util.getSystemErrorName() instead.',
    'DEP0119');
```